lib/hammer_cli_gutterball/content_report.rb

```diff
@@ -0,0 +1,73 @@
+module HammerCLIGutterball
+  class ContentReportsCommand < Command
+    resource :content_reports
+    module ReportPrintable
+      def adapter
+        :csv
+      end
+
+      def execute
+        if option_async?
+          super
+        else
+          request = send_request
+          task_id = request.is_a?(Hash) ? request['id'] : request
+          task_progress(request)
+          taskoutput = load_task(task_id)
+          if taskoutput['output'] && taskoutput['output']['report_data']
+            print_data(taskoutput['output']['report_data'])
+          end
+          HammerCLI::EX_OK
+        end
+      end
```

Comment:
why is this execute override needed? i thought adding `include HammerCLIForemanTasks::Async` took care of the asynchronous bit.